cmd/kube-controller-manager/app/options/nodelifecyclecontroller.go

```diff
@@ -39,13 +39,10 @@ func (o *NodeLifecycleControllerOptions) AddFlags(fs *pflag.FlagSet) {
 		"Amount of time which we allow running Node to be unresponsive before marking it unhealthy. "+
 			"Must be N times more than kubelet's nodeStatusUpdateFrequency, "+
 			"where N means number of retries allowed for kubelet to post node status.")
-	fs.DurationVar(&o.PodEvictionTimeout.Duration, "pod-eviction-timeout", o.PodEvictionTimeout.Duration, "The grace period for deleting pods on failed nodes.")
```

Comment:
We didn't mark `--pod-eviction-timeout` as deprecated in v1.25, do we have to? It's used by taint manager.

Reply:
oh no :(

Yes, we should have.

We need to keep this as deprecated for at least one more release, but we can simply ignore it's value.

Reply:
Got it. Thanks for explaining. I'll update in the previous PR.